PythonPonies/bangazon_cli
# Adding Bangazon database file & SQL file

# Description
This pull request proposes adding two files to the repository: 
- bangazon.db
- populatedb.sql

The database contains all necessary tables for us to persist data pertaining to the Bangazon Command Line Interface.

## Number of Fixes
0 fixes. 2 new files. 

## Related Ticket(s)
#1 + #19 

## Problem to Solve
Per the product ticket requirements, we needed to have a way to persist data related to customers, products and orders associated with the Bangazon Command Line Interface. This database provides the framework for which the data can be persisted. 

## Expected Behavior
The database should be able to be viewed and used to store necessary data. 

## Steps to Test Solution

1.  git fetch
2. git checkout swh-sql-db
3. open bangazon.db in db Browser
4. ensure that all necessary tables are present in correct structure

## Testing

[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[ ] I certify that all existing tests pass

## Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods

## Deploy Notes
N/A
